src/Maestro/Maestro.Web/Startup.cs

```diff
@@ -240,6 +242,28 @@ public void ConfigureServices(IServiceCollection services)
             );
 
             services.AddMergePolicies();
+
+
+            // Configure access to Azure App Configuration
+            string appConfigEndpointUri = Configuration["AppConfigurationUri"];
+            ConfigurationBuilder builder = new ConfigurationBuilder();
+            TokenCredential credential = appConfigEndpointUri.Contains("maestrolocal") ? 
+                new DefaultAzureCredential() : 
+                (TokenCredential) new ManagedIdentityCredential();
+
+            builder.AddAzureAppConfiguration(options =>
+            {
+                options.Connect(new Uri(appConfigEndpointUri), credential)
+                    .ConfigureRefresh(refresh =>
+                    {
+                        refresh.Register("AutoBuildPromotion", "Maestro")
+                            .SetCacheExpiration(TimeSpan.FromSeconds(1));
```

Comment:
Does this mean we will poll Azure for this value every second? Is such a low polling interval to the configuration store desirable?

Reply:
Ahh nevermind, I see now, this only happens when you call Refresh(), so this won't poll until we actually try to create a Build object.